PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/rendering/bidding/data/bid/BidResponse.java

```diff
@@ -223,6 +219,10 @@ public boolean isVideo() {
         return false;
     }
 
+    public boolean withNoBids() {
```

Comment:
Let's remove this method and encourage the usage of `getWinningBid() == null` if needed. 

In addition, the absence of a winning bid doesn't always mean "no bids". I'm not pretty sure about the PBS behavior, but it may happen that bidder responds with a bid that does meet some requirements, and can't be marked as a winning bid. So the response will contain bids, but no winning bid.